### ISSUE TITLE
Fluentbit stops sending logs to FluentD via TLS after upgrade from 1.2 to 1.3

### ISSUE THREAD

**Issue by konstantin-kornienko** [fluent/fluent-bit#1620]
## Bug Report

**Describe the bug**
Fluentbit 1.2.x was sending to FluentD without a problem.
Fluentbit config:
```
[OUTPUT]
  Name              forward
  Match             source.*
  Host              fluentd
  Port              24224
  tls               On
  tls.verify        On
  tls.ca_file       /etc/td-agent-bit/ca.crt
  tls.crt_file      /etc/td-agent-bit/client.crt
  tls.key_file      /etc/td-agent-bit/client.key
  Shared_Key        SHAREDKEY
```

FluentD v3.5.0 config:
```
<source>
  @id forward
  @type forward
  port 24224
  <security>
    self_hostname fleuntd
    shared_key SHAREDKEY
  </security>
  <transport tls>
    version TLSv1_2
    ca_path /etc/td-agent/ca.crt
    cert_path /etc/td-agent/server.crt
    private_key_path /etc/td-agent/server.key
    client_cert_auth true
  </transport>
</source>
```

**Expected behavior**
Before update to fluentbit 1.3.x everything worked fine.
After upgrade we have in fluentbit logs:
```
[error] [out_fw] Failed PING
[error] [out_fw] server requires username and password
[error] [out_fw] Failed PING
[error] [out_fw] server requires username and password
[error] [out_fw] Failed PING
[error] [out_fw] server requires username and password
[error] [out_fw] Failed PING
[error] [out_fw] server requires username and password
[error] [out_fw] Failed PING
[error] [out_fw] server requires username and password
[error] [out_fw] Failed PING
```
**Screenshots**

**Your Environment**
* Operating System and version: Ubuntu 18
* Filters and plugins: no

**Comment by bluebike** [comment]
Duplicate of #1612 which is solved and closed.

**Comment by edsiper** [comment]
Thanks everyone.

The fix will be part of v1.3.2 release this week.
